getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -9,24 +9,33 @@ The following steps detail what is needed to setup the SDK and the engine.
 Download the Android SDK
 ------------------------
 
-Download and install the Android SDK from
-`developer.android.com <https://developer.android.com/studio/>`__.
+Download and install the Android SDK. 
 
-If you install Android Studio, you need to run it once to complete the SDK setup.
+- You can install it using `Android Studio <https://developer.android.com/studio/>`__.
+  
+  - Run it once to complete the SDK setup using these `instructions <https://developer.android.com/studio/intro/update#sdk-manager>`__.
+  - Ensure that the `required packages <https://developer.android.com/studio/intro/update#recommended>`__ are installed as well.
 
-Install OpenJDK 8
+- You can install it using the `command line tools <https://developer.android.com/studio/#command-tools>`__.
+  
+  - Once the command line tools are installed, run the `sdkmanager <https://developer.android.com/studio/command-line/sdkmanager>`__ command to complete the setup process:
+
+::
+
+    sdkmanager --sdk_root=<android_sdk_path> "platform-tools" "build-tools;30.0.1" "platforms;android-29" "cmdline-tools;latest"
+
+Install OpenJDK
```

Comment:
That's no longer needed in 3.2.4 no? We needed OpenJDK for jarsigner.

Reply:
java still needs to be installed on the machine. That is usually covered by installing `Android Studio`, but if the user goes through the command line route, it's an additional step, so I decided to leave the `OpenJDK` section to cover all bases.

At the very least, we're no longer limited to version 8.

Reply:
Only for gradle / custom build then I guess?

It should likely call out that it should be OpenJDK version 8 or higher, to clarify that higher versions are supported (as done in https://gradle.org/install/). Many users have the memory that JDK 8 and only 8 is necessary, so it's good to make sure that updated documentation does not just remove the number but instead make clear that higher releases now work.

Reply:
> Only for gradle / custom build then I guess?

Let's go with this as is as I want to sync the translations with the latest docs. Since we seem to be moving towards custom builds being the default workflow eventually, it's likely good to ensure that people have Java installed anyway.